src/stylesheets/includes/_not-sorted-scss.scss

```diff
@@ -119,3 +119,9 @@ em {
 	max-height:220px;
 }
 
+// fix the bug that mouse cursor changes on button when using edge browser - we have to set it explicitly 
+// because it derives some default browser functionality
+button{
```

Comment:
Please use a more specific selector then html button.
This has too many implications all over the site. 

Target just the specific problematic button

Reply:
ALL player buttons are problematic when edge is used